front/src/common/Map/Layers/Errors.ts

```diff
@@ -1,102 +1,107 @@
 import { CircleLayer, LineLayer, SymbolLayer } from 'react-map-gl/maplibre';
 
-import { Theme, OmitLayer } from 'types';
+import { allInfraErrorTypes } from 'applications/editor/components/InfraErrors/types';
+import { OmitLayer } from 'types';
+import { LayerContext } from './types';
 
-export function getLineErrorsLayerProps(params: {
-  colors: Theme;
-  sourceTable?: string;
-}): OmitLayer<LineLayer> {
+const LINE_OBJECT = ['TrackSection', 'Catenary', 'SpeedSection'];
+export function getLineErrorsLayerProps(context: LayerContext): OmitLayer<LineLayer> {
+  const allTypes = context.issuesSettings?.types
```

Comment:
allTypes can be confusing with allInfraErrorTypes
```suggestion
  const errorTypes = context.issuesSettings.types
```

Reply:
You did not change this